### PR TITLE
Added: Multimedia - Pragma

### PR DESCRIPTION
**Pull request by MuhammadAly11**
### What is your contribution type?

- [x] Suggest an Awesome item.

### What's in this PR?

Adding new multimedia channel: Pragma #8 

### Checklist

- [x] Ensure to check our [Contribution Criteria](../blob/main/CONTRIBUTING.md#criteria).
- [x] The contribution doesn't already exist (you can use [search functionality](https://awesome-arabic-speakers.dev/search/)).
- [x] The contribution is provided in both Arabic and English.

### CODE REVIEW

File: content/post/multimedia/channel/pragma/index.ar.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Pragma قناة
+description: القناة عن صناعة البرمجيات ، والكود النظيف ، وتصميم البرامج، والأدوات.
+slug: pragma
+date: 2024-09-08 00:00:00+0000
+image: pragma.png
```
Comment (aabouzaid) [inline content/post/multimedia/channel/pragma/index.ar.md:6]: Let's find another cover. The image dimensions will not show correctly.
It should be something like 1024 x 768

Comment (MuhammadAly11) [reply content/post/multimedia/channel/pragma/index.ar.md:6]: The closest I can get is 1024x576, after using ```magick``` to modify dimensions of it's Facebook group cover keeping the aspect ratio.

Comment (MuhammadAly11) [reply content/post/multimedia/channel/pragma/index.ar.md:6]: I pushed another cover with right dimensions

Comment (aabouzaid) [reply content/post/multimedia/channel/pragma/index.ar.md:6]: It's good enough, thank you :raised_hands:

**Review by aabouzaid — APPROVED** [review]

**Pull request merged**